### DISCUSSION TITLE
UC work without headless but not work with headless option ON?

### DISCUSSION THREAD

**Discussion by JohnDoe15152** [ultrafunkamsterdam/undetected-chromedriver#1134] [Q&A]
Hello, I have been using undetected chromedriver for a few months with my scripts. Recently the headless mode has stopped working, while with non-headless everything works. Does anyone know why and how to fix it please? Thanks for the work and have a nice day!

**Comment by dfgrisales5078** [comment]
@JohnDoe15152 I have the same issue, it works except when I enable the headless option.

**Comment by dijikul** [comment]
I am experiencing the same issue.

The sample code runs:

```
import logging
logging.basicConfig(level=10)
import undetected_chromedriver as uc
driver = uc.Chrome()
driver.get('https://nowsecure.nl') # or any other website
```


But when adding headless=True to the driver as such: 
```
import logging
logging.basicConfig(level=10)
import undetected_chromedriver as uc
driver = uc.Chrome(headless=True)
driver.get('https://nowsecure.nl') # or any other website
```

The following occurs:

![image](https://user-images.githubusercontent.com/2670528/235560717-75057b21-d1d1-4e78-93bd-413f58c23e0f.png)

Chromedriver is version `112.0.5615.49` to align with Chrome `112.0.5615.138`.

**Comment by benethor** [comment]
Try setting window size explicitly for headless mode(as in, add `--window-size=1920,1080` to options). "New Headless" mainlined in Chrome 110 sets default viewport for headless windows to 0x0 pixels.
